packages/main/src/themes/RatingIndicator.css

```diff
@@ -0,0 +1,37 @@
+:host(:not([hidden])) {
+	display: inline-block;
+	font-size: 1.5rem;
+	cursor: pointer;
+}
+
+:host([disabled]),
+:host([read-only]) {
+	opacity: .8;
+	cursor: initial;
+	outline: none;
+}
+
+:host([size="Large"]) {
+	font-size: 2rem;
+}
+
+:host([_focused]) {
+	outline: 1px dotted var(--sapContent_FocusColor);
+}
+
+.ui5-rating-indicator-root {
+	outline: none;
+}
+
+.icon {
```

Comment:
It is good to name this class by the convention: "ui5-rating-indicator-icon". 
And, If you want to minify the classes you can rename everything to ui5-ri-root, ui5-ri-stars,  ui5-ri-icon,  ui5-ri-icon--active, but this is only a suggestion.